easybuild/easyconfigs/c/cutadapt/cutadapt-3.4-GCCcore-9.3.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,56 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman, Jonas Demeulemeester
+# The Francis Crick Institute
+# Modufied by: Albert Bogdanowicz
+# Institute of Biochemistry and Biophysics PAS
+# 3.4
+# Erica Bianco (HPCNow!)
+
+easyblock = 'PythonBundle'
+
+name = 'cutadapt'
+version = '3.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://opensource.scilifelab.se/projects/cutadapt/'
+description = """Cutadapt finds and removes adapter sequences, primers, poly-A tails and
+ other types of unwanted sequence from your high-throughput sequencing reads."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+# exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('xopen', '1.1.0', {
```

Comment:
@kErica `pip check` is failing because `xopen` requires `isal` - please add that extension before `xopen`.